pom.xml

```diff
@@ -35,4 +35,45 @@
          <scope>test</scope>
       </dependency>
    </dependencies>
+  <build>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-jar-plugin</artifactId>
+            <version>2.2</version>
+            <!-- nothing here -->
+        </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <version>2.2-beta-4</version>
+            <configuration>
+              <archive>
+                <manifest>
+                  <mainClass>com.csc530.sat.App</mainClass>
```

Comment:
Is there an easier way to do this I'm not super familiar with maven but I don't remember it being this ridiculous just to make a jar